template/src/components/ui/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,67 @@
+import { Button, IconButton, styled } from "@mui/material";
+import * as React from "react";
+import { CustomDialog } from "./CustomDialog";
+import { Icon } from "./Icon";
+
+const HeaderContainer = styled("div")({
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+});
+
+const MessageContainer = styled("div")({
+    textAlign: "left",
+});
+
+const ButtonContainer = styled("div")({
+    display: "flex",
+    alignItems: "center",
+    gap: 16,
+});
+
+const CancelLabel = styled("p")({
+    fontWeight: 600,
+    lineHeight: "16px",
+});
+
+export const ConfirmationDialog = ({
+    open,
+    title,
+    message,
+    submitLabel,
+    cancelLabel,
+    onSubmit,
+    onCancel,
+    hideCancel,
+}: {
+    open?: boolean;
+    title: string;
+    message: React.ReactNode;
+    submitLabel?: string;
+    cancelLabel?: string;
+    onSubmit: () => void;
+    onCancel: () => void;
+    hideCancel?: boolean;
+}) => {
+    return (
+        <CustomDialog open={open}>
+            <HeaderContainer>
+                <h1 style={{ fontSize: 32 }}>{title}</h1>
+                <IconButton onClick={onCancel}>
+                    <Icon name="close" />
+                </IconButton>
+            </HeaderContainer>
+            <MessageContainer style={{ marginTop: 16 }}>{message}</MessageContainer>
+            <ButtonContainer style={{ marginTop: 32 }}>
+                {!hideCancel && (
+                    <Button onClick={onCancel} style={{ backgroundColor: "transparent" }}>
+                        <CancelLabel>{cancelLabel}</CancelLabel>
```

Comment:
CancelLabel is a `<p>` tag which is invalid in a `<button>`. Please switch the label to a `<span>` or better remove the CancelLabel completely because I think it isn't necessary.

Reply:
[remove messagecontainer](https://github.com/allaboutapps/cra-template-aaa-typescript/pull/23/commits/ab63ab684389f130425c33896cfdc3e1fa616b32)